extensions/applicationinsights-clickanalytics-js/src/handlers/AutoCaptureHandler.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * @copyright Microsoft 2020
+ */
+
+import { IDiagnosticLogger, _InternalMessageId, getWindow, getDocument, EventHelper } from "@microsoft/applicationinsights-core-js";
+import { IAutoCaptureHandler, IPageActionOverrideValues, IClickAnalyticsConfiguration } from '../Interfaces/Datamodel'
+import { isRightClick, isLeftClick, isKeyboardEnter, isKeyboardSpace, isMiddleClick, isElementDnt } from '../common/Utils';
+import { ActionType } from '../Enums';
+import { ClickAnalyticsPlugin } from "../ClickAnalyticsPlugin";
+import { PageAction } from '../events/PageAction';
+
+const clickCaptureInputTypes = { BUTTON: true, CHECKBOX: true, RADIO: true, RESET: true, SUBMIT: true };
+
+export class AutoCaptureHandler implements IAutoCaptureHandler {
+    
+    /**
+     * @param analyticsPlugin - WebAnalytics plugin
+     * @param traceLogger - Trace logger to log to console.
+     */
+   constructor(protected _analyticsPlugin: ClickAnalyticsPlugin, protected _config: IClickAnalyticsConfiguration, protected _pageAction: PageAction,
+     protected _traceLogger: IDiagnosticLogger) {
+
+    }
+    // handle automatic event firing on user click
+    public click() {
+        let win = getWindow();
+        let doc = getDocument();
+        if (win) {
+            // IE9 onwards addEventListener is available, 'click' event captures mouse click. mousedown works on other browsers
+            const event = (navigator.appVersion.indexOf('MSIE') !== -1) ? 'click' : 'mousedown';
+            EventHelper.Attach(win, event , (evt:any) => { this._processClick(evt); });
+            EventHelper.Attach(win, 'keyup' , (evt:any) => { this._processClick(evt); });
+        } else if (doc && (doc as any).attachEvent) {
```

Comment:
the doc version should also use EventHelper.Attach, or you could replace both of these with CoreUtils.addEventHandler() which does this (as well as attempting to add the event to body (if present) -- from memory this is for old Safari